pydra/engine/node.py

```diff
@@ -242,23 +243,34 @@ def cache_dir(self):
     @cache_dir.setter
     def cache_dir(self, location):
         if location is not None:
-            self._cache_dir = Path(location)
+            self._cache_dir = Path(location).resolve()
```

Comment:
don't know why this would be the case.

Reply:
I'm also not sure... Can debug more, it could be something in `pathlib`. You can try to run on your laptop `test_task_nostate_1_cachedir_relativepath` without mychanges

Reply:
you can check `odir` before and after this line:https://github.com/satra/pydra/blob/005d41e7dd605103a144dcb990f933f4a0897790/pydra/engine/node.py#L299